### PR TITLE
week 14 of 2021 (April 4th - April 9st)

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

File: content/posts/April-2021.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  synchronize the suite to Fedora dist-git
+  ([packit#1192](https://github.com/packit/packit/pull/1192)).
+- Franta fixed a bug in packit which kept only appending targets to an existing
+  COPR project which is no longer a case - dropped targets are now being
+  removed
+  ([packit#1197](https://github.com/packit/packit/pull/1197)).
+- Laura fixed how packit-service pushes alerts to sentry, which should make it
+  more convenient for us, maintainers to know which project is affected by the
+  particular alert
+  ([packit-service#1048](https://github.com/packit/packit/pull/1048)).
```
Comment (jpopelka) [inline content/posts/April-2021.md:20]: The url is wrong, correct is https://github.com/packit/packit-service/pull/1048
And I'd not put it into the blog, this is not something readers care about.

Comment (TomasTomecek) [reply content/posts/April-2021.md:20]: well, I wanted to be transparent; ok, will drop it

**Comment by csomh** [comment]
regate

**Comment by csomh** [comment]
recheck

**Comment by csomh** [comment]
regate

**Comment by csomh** [comment]
regate

**Comment by TomasTomecek** [comment]
Thanks for merging this, I forgot about it.

**Pull request merged**